platform/ui/src/components/DateRange/DateRange.tsx

```diff
@@ -52,6 +32,27 @@ const DateRange = props => {
   const { id, onChange, startDate, endDate } = props;
   const [focusedInput, setFocusedInput] = useState(null);
   const renderYearsOptionsCallback = useCallback(renderYearsOptions, []);
+  const { t } = useTranslation('DatePicker');
+  const today = moment();
+  const lastWeek = moment().subtract(7, 'day');
+  const lastMonth = moment().subtract(1, 'month');
+  const studyDatePresets = [
+    {
```

Comment:
Should we add `'Today'`, `'Last 7 days'` and `'Last 30 days'` to the various DatePicker.json files? Again, just the ones you are comfortable with.